client/components/Editor/utils/changes.ts

```diff
@@ -1,14 +1,13 @@
+import { Node, Schema } from 'prosemirror-model';
 import { Step } from 'prosemirror-transform';
-import firebase from 'firebase-admin';
 import { uncompressStepJSON } from 'prosemirror-compress-pubpub';
 
-import { editorSchema } from '../firebaseAdmin';
-
 export const getStepsInChangeRange = async (
 	branchRef: firebase.database.Reference,
```

Comment:
Is `firebase` here in the global namespace?

Reply:
Yeah. I actually don't know why but TypeScript seems to accept it — and ultimately this may be our only option since writing:

```
import firebase from 'firebase';
```

throws a warning on the client recommending that we don't do this and use `firebase/app` instead. Likely it's dragging in a bunch of Firebase features that we don't use.

Reply:
Interesting. Might be worth an explicit comment hinting at something there (or even just pointing back to this PR) in case we do bump into firebase weirdness at some point.